introduction.md

```diff
@@ -0,0 +1 @@
+Hello I am a new dojo initiate. Please teach me sensei. Domo arigato gozaimasu.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.